join.spec.md

```diff
@@ -131,48 +131,42 @@ Documents MUST define a {join__Graph} enum. Each enum value defines a subgraph.
 
 The {join__Graph} enum is used as input to {@join} directives which link fields and types to subgraphs.
```

Comment:
I think we should say that each enum value MUST have a `@join__graph` directive applied to it?

Reply:
Fixed on my branch